app/models/audio_version.rb

```diff
@@ -20,4 +25,33 @@ def length(reload=false)
   def self.policy_class
     StoryAttributePolicy
   end
+
+  def compliant_with_template?
+    file_errors.nil?
+  end
+
+  private
+
+  def validate_on_template
+    return unless audio_version_template
+
+    noncompliant_file = audio_files.find do |af|
+      !af.compliant_with_template? || af.status == INVALID
```

Comment:
Won't an `INVALID` file always be non-compliant?  Or am I missing something?

Also ... does this need to account for the other incomplete audio statuses?  (NOTFOUND, VALIDATING, FAILED).  Wondering if we need to propagate those statuses up as well.